core/core-configuration/src/main/webapp/WEB-INF/conf/wcm-core/core-connector-configuration.xml

```diff
@@ -132,5 +132,60 @@
       <description>Add Dynamic Skin Plugins into FCK editor </description>
     </component-plugin>
   </external-component-plugins>
+  
+  <!-- #################################################### DLP #################################################### -->
+  
+  <external-component-plugins>
+    <target-component>org.exoplatform.services.jcr.impl.ext.action.SessionActionCatalog</target-component>
+    <component-plugin>
+      <name>addActions</name>
+      <set-method>addPlugin</set-method>
+      <type>org.exoplatform.services.jcr.impl.ext.action.AddActionsPlugin</type>
+      <description>add actions plugin</description>
+      <init-params>
+        <object-param>
+          <name>actions</name>
+          <object type="org.exoplatform.services.jcr.impl.ext.action.AddActionsPlugin$ActionsConfig">
+            <field name="actions">
+              <collection type="java.util.ArrayList">
+                <value>
+                  <object type="org.exoplatform.services.jcr.impl.ext.action.ActionConfiguration">
+                    <field name="workspace"><string>collaboration</string></field>
+                    <field name="eventTypes"><string>addNode,addProperty,changeProperty,removeProperty</string></field>

```

Comment:
Why we should launch the action on removeProperty event ?

Reply:
Removed